### DISCUSSION TITLE
FSI of vocal folds

### DISCUSSION THREAD

**Discussion by maxnezdyur** [KratosMultiphysics/Kratos#9832] [Q&A] [answered]
Are there any examples of using the embedded framework with contact? The VF undergo large deformation and more and more I see that the body fitted ALE based approach is becoming unsustainable.

**Comment by loumalouomega** [comment]
@rubenzorrilla our pending stuff...

**Comment by rubenzorrilla** [comment, marked as answer]
Hi @maxnezdyur. We did some preliminary cases (with promising results) it as @loumalouomega is pointing, however there are no examples in the repo. In any case, it should be "not so difficult" to set it up if you are familiar with Kratos. I'd suggest starting from any of the embedded FSI cases in the Examples repository. Then, in the structure solver settings you should add the contact ones (you can find examples in any of the contact cases in the examples repo).

**Reply by maxnezdyur** [reply]
I am just learning Kratos, so I apologize for any naive questions. Is there a way to get an updated example of "Mixer with flexible blades"? There seems to be a few conflicts with that example and the 9.x versions of Kratos. I tried to update any deprecated settings, only to get a simulation that diverges and crashes after a few time steps(orders of magnitude less than the original). Is there any more documentation also, in general? I wasn't sure if there was a resource I was missing that gave more information about setting up different simulations and the settings required and their meanings.

**Reply by rubenzorrilla** [reply]
Hi @maxnezdyur! I did a fast try and the settings seem to be a bit outdated as you point. I'll try to update them ASAP (sorry for the inconveniences). The pity is that we don't have any user manual as such, so the documentation is somehow the combination of the code comments, the GitHub wiki and the scientific papers published with Kratos.

I'll keep you updated :wink:

**Reply by maxnezdyur** [reply]
Thank you for taking the time!

**Reply by maxnezdyur** [reply]
@rubenzorrilla  Any update on this? Thanks!

**Reply by rubenzorrilla** [reply]
Hi @maxnezdyur. Yes! Basically I've been doing some forensic informatics to compare the current with the old version of Kratos. The issue raised up when implementing the block Newton equations coupling. I introduced some changes that made the solver slightly less robust. I'm currently working on it so it should be ready for you to try soon.

**Reply by rubenzorrilla** [reply]
@maxnezdyur see #9881. With this the embedded mixer example is working as it used to.

**Reply by maxnezdyur** [reply]
Thank you for all the help. I will look at it when I get a chance!

**Reply by rubenzorrilla** [reply]
Thank you for reporting! I'll also create a PR in the examples repository with the updated settings.

**Reply by rubenzorrilla** [reply]
@maxnezdyur this is just to let you know that I've merged #9881, so you should be able to run the flexible mixer example with current master branch out of the box.

**Comment by RiccardoRossi** [comment]
mentioning  @AlejandroCornejo who also has expertise about contact
